src/app/config/reducer.js

```diff
@@ -674,6 +674,16 @@ export default function reducer(state = initialState, action) {
                 },
             };
         }
+        case groupsTypes.DELETE_GROUP_SUCCESS: {
+            return {
+                ...state,
+                groups: {
+                    ...state.groups,
+                    all: state.groups.all.filter(group => group.id !== action.id),
```

Comment:
Questions above `[src/app/config/groups/thunks.js](https://github.com/ONSdigital/florence/pull/703/files#diff-428c5e7b6cd34850a6563eb2f1bbc9c8d01673d835f8a8ac637a6d16b2280bd5)` line 79 refers to this always being an array even if empty.